src/main/java/core/basesyntax/services/ReadFromFileService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.services;
+
+import java.util.List;
+
+public interface ReadFromFileService {
+    List<String> getDbFromFile(String fileName);
```

Comment:
```suggestion
    List<String> readFromFile(String fileName);
```

be attentive to method naming, this method just read data from file, we don't get any db from it